kubernetes/services/deleteYaml.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Install prerequisites
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+
+  echo "delete yaml files"
+  
+  kubectl delete clusterrolebinding ignite-alcor
+  kubectl delete clusterrole ignite-alcor
+  #kubectl delete namespaces ignite-alcor
```

Comment:
Consider using same ns for all ignite pods/containers, regardless of which services that are connected to. This is not priority.

Reply:
Sure